src/app/routes/connector-ui/contract-agreement-page/contract-agreement-cards/contract-agreement-cards.component.html

```diff
@@ -58,18 +59,35 @@
           </div>
         </div>
       </div>
-      <!-- Counterparty Address -->
-      <div class="flex flex-col flex-grow">
-        <div class="text-slate text-sm uppercase tracking-wider">
-          Other Connector
+      <div class="flex flex-row space-x-[10px] justify-between">
+        <!-- Counterparty Address -->
+        <div class="flex flex-col">
+          <div class="text-slate text-sm uppercase">Other Connector</div>
+          <div>
+            <a
+              class="link break-all"
+              target="_blank"
+              [attr.href]="card.counterPartyAddress"
+              >{{ card.counterPartyAddress }}</a
+            >
+          </div>
         </div>
-        <div>
-          <a
-            class="link"
-            target="_blank"
-            [attr.href]="card.counterPartyAddress"
-            >{{ card.counterPartyAddress }}</a
-          >
+        <!-- Status -->
+        <div
+          *ngIf="card.isConsumingLimitsEnforced"
+          class="flex flex-col"
+          [matTooltip]="card.statusTooltipText">
+          <div class="text-slate text-sm uppercase tracking-wider text-right">
+            Status
+            <mat-icon
+              *ngIf="card.statusText === 'Inactive'"
+              class="text-warn mat-icon-[10px]"
+              >warning
```

Comment:
I'll have to check this in the final visual test later, but is it really good if we use a warning icon? In general, us disabling older consuming contracts will happen, and be a matter of course. It might be hard to spot at first, but I am not sure if it's an error state or something that requires an action from the user.